dev/how-to/deploy/orchestrated/ansible.md

```diff
@@ -119,36 +119,16 @@ The key's randomart image is:
 
 ## 在中控机器上下载 TiDB-Ansible
 
-以 `tidb` 用户登录中控机并进入 `/home/tidb` 目录。以下为 tidb-ansible 与 TiDB 的版本对应关系，版本选择可以咨询官方。
+以 `tidb` 用户登录中控机并进入 `/home/tidb` 目录。使用以下命令从 [TiDB-Ansible 项目](https://github.com/pingcap/tidb-ansible)上下载 master 分支的 TiDB-Ansible，默认的文件夹名称为 `tidb-ansible`。
 
-| TiDB 版本 | tidb-ansible tag | 备注 |
-| -------- | ---------------- | --- |
-| 2.0 版本 | v2.0.10、v2.0.11 | 最新 2.0 稳定版本，可用于生产环境。 |
-| 2.1 版本 | v2.1.1 ~ v2.1.8 等 | 最新 2.1 稳定版本，可用于生产环境（建议）。 |
-| 3.0 版本 | v3.0.0-beta、v3.0.0-beta.1 等 | 目前是 beta 版本，不建议用于生产环境。 |
-| latest 版本 | None | 包含最新特性，每日更新，不建议用于生产环境。 |
-
-使用以下命令从 Github [TiDB-Ansible 项目](https://github.com/pingcap/tidb-ansible)上下载 TiDB-Ansible [相应版本](https://github.com/pingcap/tidb-ansible/tags)，默认的文件夹名称为 `tidb-ansible`。
-
-> **注意：**
->
-> 部署和升级 TiDB 集群需使用对应的 tidb-ansible 版本，通过改 `inventory.ini` 文件中的版本来混用可能会产生一些错误。
-
-- 下载指定 tag 的 tidb-ansible：
-
-    ```
-    $ git clone -b $tag https://github.com/pingcap/tidb-ansible.git
-    ```
-
-- 下载 latest 版本对应的 tidb-ansible：
-
-    ```
-    $ git clone https://github.com/pingcap/tidb-ansible.git
-    ```
+```
+$ git clone https://github.com/pingcap/tidb-ansible.git
+```
```

Comment:
@dcalvin What do you think of removing description about previous versions?

Reply:
I think it's the right thing to do. Not only making the doc shorter, but also adaptive the specific version